### PR TITLE
Edit on GitHub

### PR DESCRIPTION
**Pull request by RolandCsibrei**

### CODE REVIEW

**Comment by RaananW** [comment]
There is a build issue - is it possible that you didn't push an update to one of the interfaces?

Also - I noticed it's absolute positioned, would it be possible to put it at the end of the file, bottom right side?

Thanks!

**Comment by RolandCsibrei** [comment]
Hello!
I am sorry! I didn't have any build issues. I will double check. I thought
, you will reposition it, to fit your expectations (maybe I've
misunderstood your message on the forum) but sure I can move it to the end
of each page, no problem. I made it this way, because I was wondering
whether we can navigate the user directly to the current text block being
displayed in the docs. In this case it wouldn't make sense to have only at
the bottom.

File: pages/[...id].tsx [hunk 1]
```diff
@@ -143,14 +148,21 @@ export const DocumentationPage: FunctionComponent<IDocumentationPageProps> = ({
...
+                                    {/* Assuming video overview sis always youtube! Can be changed */}
                                     <MediaMarkdownComponent url={metadata.videoOverview} type="youtube"></MediaMarkdownComponent>
                                 </>
                             )}
                             {renderedContent}
                             {metadata.videoContent && <VideoCollection videoLinks={metadata.videoContent}></VideoCollection>}
                             <BucketContent title="Further reading" childPages={relatedArticles} externalLinks={relatedExternalLinks}></BucketContent>
                             <BucketContent childPages={childPages}></BucketContent>
+                            <div id="edit-on-github">
+                                <Tooltip title={`Edit on GitHub`} aria-label="Edit on GitHub">
```
Comment (RaananW) [inline pages/[...id].tsx]:
```suggestion
                                <Tooltip title={`Edit this page on GitHub`} aria-label="Edit this page on GitHub">
```

File: pages/[...id].tsx [hunk 2]
```diff
@@ -143,14 +148,21 @@ export const DocumentationPage: FunctionComponent<IDocumentationPageProps> = ({
                             {metadata.videoOverview && (
                                 <>
                                     <h2>Video Overview</h2>
-                                    {/* Assuming video overview is always youtube! Can be changed */}
+                                    {/* Assuming video overview sis always youtube! Can be changed */}
```
Comment (RaananW) [inline pages/[...id].tsx]:
```suggestion
                                    {/* Assuming video overview is always youtube! Can be changed */}
```

**Review by RaananW — APPROVED** [review]
Just nits. merging right after :-)

**Pull request merged**